homeassistant/components/tado/__init__.py

```diff
@@ -215,7 +215,7 @@ def update_sensor(self, sensor_type, sensor):
             else:
                 _LOGGER.debug("Unknown sensor: %s", sensor_type)
                 return
-        except RuntimeError:
+        except:
```

Comment:
In my opinion, catching all exceptions broadly is a bad practice.
I think it is better if the upstream package handles the possible exceptions better and would throw specific errors we can handle correctly.

Have you considered contributing an improvement upstream?

Reply:
> In my opinion, catching all exceptions broadly is a bad practice.

If you want I can update the PR catching only `ConnectionError`.

> I think it is better if the upstream package handles the possible exceptions better and would throw specific errors we can handle correctly.

Sure, but that's up to the upstream maintainers.

> Have you considered contributing an improvement upstream?

Unfortunately I don't have the time to do that, so I've created an issue upstream.